### ISSUE TITLE
`history-size` limited to 10 entries

### ISSUE THREAD

**Issue by Anuskuss** [SUPERCILEX/gnome-clipboard-history#182]
### Describe the bug

I have to limit the `history-size` to 10 otherwise the window won't open.

### How To Reproduce

1. Set the `history-size` to 30 and create 30 entries
2. Make 11 favorites
3. The 12th favorite closes the window and it won't reopen again unless you clean the history until it's only 10 elements (e.g. by setting `history-size` to 10 and copying something)

#### Versions

```
GNOME Shell 46.4
GCH   Version: 44
org.gnome.shell.extensions.clipboard-history cache-only-favorites false
org.gnome.shell.extensions.clipboard-history cache-size 100
org.gnome.shell.extensions.clipboard-history clear-history @as []
org.gnome.shell.extensions.clipboard-history confirm-clear true
org.gnome.shell.extensions.clipboard-history disable-down-arrow true
org.gnome.shell.extensions.clipboard-history display-mode 0
org.gnome.shell.extensions.clipboard-history enable-keybindings false
org.gnome.shell.extensions.clipboard-history history-size 30
org.gnome.shell.extensions.clipboard-history ignore-password-mimes true
org.gnome.shell.extensions.clipboard-history move-item-first true
org.gnome.shell.extensions.clipboard-history next-entry @as []
org.gnome.shell.extensions.clipboard-history notify-on-copy false
org.gnome.shell.extensions.clipboard-history paste-on-selection false
org.gnome.shell.extensions.clipboard-history prev-entry @as []
org.gnome.shell.extensions.clipboard-history private-mode false
org.gnome.shell.extensions.clipboard-history process-primary-selection false
org.gnome.shell.extensions.clipboard-history strip-text false
org.gnome.shell.extensions.clipboard-history toggle-menu ['<Super><Shift>V']
org.gnome.shell.extensions.clipboard-history toggle-private-mode ['<Super><Shift>P']
org.gnome.shell.extensions.clipboard-history topbar-preview-size 10
org.gnome.shell.extensions.clipboard-history window-width-percentage 25
```

**Comment by SUPERCILEX** [comment]
Wat. Are you a new user or did this bug start happening on an existing installation?

**Comment by Anuskuss** [comment]
This has been happening for forever but I decided to finally report it today.

**Comment by SUPERCILEX** [comment]
Lol, k. If you run `sudo journalctl -f /usr/bin/gnome-shell` in the terminal and then reproduce the bug, does anything pop with `clipboard-history` in it?

**Comment by Anuskuss** [comment]
No but are you implying that you can't reproduce this?

**Comment by SUPERCILEX** [comment]
Yeah, I'm not even on Gnome 46 and have switched to my replacement as per https://github.com/SUPERCILEX/gnome-clipboard-history#project-status-replaced-by-ringboard. Also nobody else has reported anything like this bug so something very weird is probably going on.

**Comment by Edu4rdSHL** [comment]
> Yeah, I'm not even on Gnome 46 and have switched to my replacement as per https://github.com/SUPERCILEX/gnome-clipboard-history#project-status-replaced-by-ringboard. Also nobody else has reported anything like this bug so something very weird is probably going on.

I know that you've been working on it, and all the advantages it has. But, for Gnome users, how do you integrate that with Gnome? There's no Gnome Extension or something for it, it's just a command line, afaik.

**Comment by Anuskuss** [comment]
Okay so I made a new discovery: When your history is full and the window takes the full height of the screen it will only show you 12.5 history and 11 favorite entries. So I think what's happening is that the 12th favorite tires to push the history container "out of the screen" because the favorites container has to always show all elements while the history container has a min height of 12 elements.
So an easy way to fix this would be to make the favorites container scrollable. And they should also both occupy 50% when they both have overflow.

**Comment by SUPERCILEX** [comment]
Ohhhhhh! Nice find! One option could be to get rid of the separate section and dynamically combine the favorites and normal list into the first page.

As for Ringboard, I didn't mean that you should switch but rather that I won't be working on this extension anymore. But if you fix this bug, I'll accept a PR. :)
